<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Archeological Artifacts Inventory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_Main">
    <item>
     <widget class="QListView" name="artifactsListView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_Controls">
      <item>
       <widget class="QGroupBox" name="groupBox_Details">
        <property name="title">
         <string>Artifact Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout_Details">
         <item row="0" column="0">
          <widget class="QLabel" name="label_Id">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Id"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_Name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_Name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_Description">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="textEdit_Description"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_Material">
           <property name="text">
            <string>Material:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_Material"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_DiscoveryDate">
           <property name="text">
            <string>Discovery Date:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="dateEdit_DiscoveryDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_Location">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_Location"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Actions">
        <item>
         <widget class="QPushButton" name="pushButton_Add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Update">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Remove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Filter">
        <property name="title">
         <string>Filter</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_Filter">
         <item>
          <widget class="QLineEdit" name="lineEdit_FilterCriteria"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ApplyFilter">
           <property name="text">
            <string>Apply Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_UndoRedo">
        <item>
         <widget class="QPushButton" name="pushButton_Undo">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Redo">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_Controls">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
